### PR TITLE
Create issue

### PR DESCRIPTION
**Pull request by zpawn**
https://app.shortcut.com/deskpro/story/78686/github-create-issue

### CODE REVIEW

File: src/pages/CreateIssue.tsx [hunk 1]
```diff
@@ -1,8 +1,124 @@
...
+
+        const newIssue = {
+            title: values.title,
+            body: values.description || "",
+            milestone: !values.milestone.value ? null : values.milestone.value,
+            ...((Array.isArray(values.assignees) && values.assignees.length > 0)
+                    ? { assignees: values.assignees }
+                    : {}
+            ),
+            // ...((Array.isArray(values.labels) && values.labels.length > 0)
```
Comment (AshleyDawson) [inline src/pages/CreateIssue.tsx:65]: remove redundant code

File: src/pages/CreateIssue.tsx [hunk 2]
```diff
@@ -1,8 +1,124 @@
...
+            //     ? { labels: values.labels }
+            //     : {}
+            // ),
+        };
+
+        return createIssueService(client, {
+            repoFullName: values.repository.value,
+            // eslint-disable-next-line
+            // @ts-ignore
```
Comment (AshleyDawson) [inline src/pages/CreateIssue.tsx]: please use correct types

**Review by AshleyDawson — CHANGES_REQUESTED** [review]
minor changes requested

File: src/pages/CreateIssue.tsx (hunk 1 above)
Comment (zpawn) [reply src/pages/CreateIssue.tsx:65]: this is not redundant code, I'll be use it when I figure out how to save the labels

File: src/pages/CreateIssue.tsx (hunk 2 above)
Comment (zpawn) [reply src/pages/CreateIssue.tsx]: done

**Review by AshleyDawson — APPROVED** [review]

**Pull request merged**